docpipeline/generate.py

```diff
@@ -311,7 +309,7 @@ def version_sort(v):
     return semver.VersionInfo.parse(v)
 
 
-def build_blobs(client, blobs, credentials):
+def build_blobs(client, blobs, credentials, project_id):
```

Comment:
We don't seem to be using `project_id` at the moment. Do we actually need it?

Reply:
Good catch! Done. We also didn't need `credentials`, so I removed that, too.